### DISCUSSION TITLE
Assertions не срабатывают

### DISCUSSION THREAD

**Discussion by choitim1** [open-tutorials/cypress#98] [Q&A]
Решил написать  на Cypress функциональный тест формы регистрации https://automationteststore.com/ из предложенного первого туториала и столкнулся с тем что:
-
```javasrcript
.should('have text',' someText ');
```
выдает ошибку, хотя текст был внесен сайпросом

- если все **it** объединить в один блок то все проходит

- сайпрос не сохраняет введенный пароль
 
``` javescript
cy.get("#AccountFrm_password").type("test1");
```

вот ссылка на код  https://github.com/choitim1/automationTestStore

**Comment by breslavsky** [comment]
У вас ошибка, пропущена точка `have.text`

```js
.should('have.text',' some text ')
```

> если все it объединить в один блок то все проходит

Какие блоки? В каком файле?

> сайпрос не сохраняет введенный пароль

```js
cy.get("#AccountFrm_password").type("test1");
```

Наверное не вводит в поле ввода? Приложите текст ошибки.
